lib/active_merchant/billing/gateways/bambora_apac.rb

```diff
@@ -72,11 +72,11 @@ def refund(money, authorization, options={})
         end
       end
 
-      def void(money, authorization, options={})
+      def void(authorization, options={})
         commit('SubmitSingleVoid') do |xml|
           xml.Void do
             xml.Receipt authorization
-            add_amount(xml, money)
```

Comment:
I like this change! Any reason you spared the other `add_amount` calls in this gateway? It seems like it'd be good to either not that method at all in here, or _only_ use it. What think ye?

Reply:
Good point! Since the method was just a single line, I went for the option of deleting the method and replacing it throughout the other transaction methods.